constraints.txt

```diff
@@ -23,6 +23,7 @@ eea.schema.slate==1.2
 eea.sentry==2.4
 eea.volto.policy==2.1
 eea.zotero==1.4
+redturtle.rssservice==2.2.0
```

Comment:
@avoinea wondering if we should also add it to `eea.kitkat`?

Reply:
@nileshgulia1 If you're going to add the frontend add-on to volto-eea-kitkat **addons** and **dependencies** sections, yes. Now as the eea.kitkat is also used within PloneSaaS, you'll have to pin it **redturtle.rssservice** also to https://github.com/eea/eea.docker.plone/blob/master/src/plone/versions.cfg